### PR TITLE
Add claim identity confirmation mailer for automated checks claim verifiers identity no match [ECP-342]

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: spec/mailers/claim_mailer_spec.rb [hunk 1]
```diff
@@ -89,6 +89,23 @@
...
+      describe "#identity_confirmation" do
+        let(:claim) { build(:claim, :submitted, policy: policy) }
+        let(:mail) { ClaimMailer.identity_confirmation(claim) }
+
+        it_behaves_like "an email related to a claim", policy
+
+        it "mentions the claimant's identity needs to be confirmed" do
+          expect(mail.subject).to include("verify your identity")
+
+          expect(mail.body.encoded).to(
```
Comment (nomasprime) [inline spec/mailers/claim_mailer_spec.rb:102]: Add expectation capturing the gist of the email body.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -69,6 +69,7 @@
   config.include GeckoboardHelpers
   config.include DqtHelpers
   config.include FixtureHelpers
+  config.include IsExpectedWithBlock
```
Comment (nomasprime) [inline spec/rails_helper.rb:72]: Should highlight this by moving to another commit.

**Review by WorkSutton — APPROVED** [review]
